### PR TITLE
Retain seconds when doing update_date_time

### PR DESCRIPTION
**Pull request by donaldpiret**

### CODE REVIEW

**Comment by einzige** [comment]
@donaldpiret can you please add specs?

**Comment by donaldpiret** [comment]
@einzige Sure, will do so tomorrow!

File: lib/date_time_attribute/container.rb [hunk 1]
```diff
@@ -111,7 +111,7 @@ def parse(val)
 
     def update_date_time
       if @date || @time
-        date_time = self.class.parser.parse("#{date.try(:strftime, '%Y-%m-%d')} #{time.try(:strftime, '%H:%M')}")
+        date_time = self.class.parser.parse("#{date.try(:strftime, '%Y-%m-%d')} #{time.try(:strftime, '%H:%M:%S')}")
```
Comment (einzige) [inline lib/date_time_attribute/container.rb:114]: shut up, dog

**Comment by donaldpiret** [comment]
Arg, damn I had completely forgotten to commit the spec for this, and getting a notification of the merge reminded me of it.
If you still want it I can open another pull request for it.

**Comment by einzige** [comment]
sure, i am always opened to it. appreciate your PRs regardless of any specs missing :)

**Pull request merged**